mumd/src/state.rs

```diff
@@ -43,18 +43,13 @@ macro_rules! now {
 
 type Responses = Box<dyn Iterator<Item = mumlib::error::Result<Option<CommandResponse>>>>;
 
+type TcpEventCallback = Box<dyn FnOnce(TcpEventData<'_>) -> Responses>;
+type TcpEventSubscriberCallback = Box<dyn FnMut(TcpEventData<'_>,&mut mpsc::UnboundedSender<mumlib::error::Result<Option<CommandResponse>>>,) -> bool>;
```

Comment:
This type is still too long IMO.

Reply:
I agree, but I don't really know how to make it shorter. It would maybe be making the `UnboundedSender<...>` it's own type, but I don't think that feels right.

Reply:
Alright. In that case, how about some line breaks?